md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java

```diff
@@ -93,6 +89,7 @@ public final CommandLine getCommandLine(String[] params, String[][] paramStructu
             return cmd;
 
         } catch (ParseException e) {
+            LOGGER.error(e.getMessage());
             //System.err cannot be removed as it is an command line argument
             printUsage("java <main_class> ", e.getMessage(), options, System.err);
             System.exit(1);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this call to "exit" or ensure it is really required. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1147)